nix/overlay.nix

```diff
@@ -67,19 +67,58 @@ with prev.lib; let
       self = luajit;
     };
 
-  neorocks = prev.pkgs.symlinkJoin {
-    name = "neorocks";
-    paths = [
-      final.luajit
-      final.luajitPackages.luarocks
-      neolua-stable-wrapper
-      neolua-nightly-wrapper
-    ];
-  };
+  mkNeorocks = neolua-pkgs:
+    prev.pkgs.symlinkJoin {
+      name = "neorocks";
+      paths =
+        [
+          final.luajit
+          final.luajitPackages.luarocks
+        ]
+        ++ neolua-pkgs;
+    };
+
+  neorocks = mkNeorocks [
+    neolua-stable-wrapper
+    neolua-nightly-wrapper
+  ];
+
+  neorocksTest = {
+    src,
+    name,
```

Comment:
For people who want to run multiple tests (e.g. with different Neovim versions), it might be good to have a `name` (for the derivation's name) and a `pname` (for the plugin's name).